addons/binding/org.openhab.binding.heos/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="heos"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>HEOS Binding</name>
+    <description>This is the binding for the Denon HEOS system.</description>
+    <author>Johannes Einig</author>
+    
+    
+    <config-description>
+ 
+        <parameter name="callbackUrl" type="text">
+            <label>Callback URL</label>
+            <description>URL to use for playing notification sounds, e.g. http://192.168.0.2:8080</description>

```

Comment:
Who's url is this openHAB or HEOS?

Reply:
This should be the URL. there your audio files are stored. 
I have to say that this is mostly copied from the Sonos binding. So I'm not absolutely sure what would be a good description.

Reply:
I tried to explain it a little bit more within the readme.md:
https://github.com/Wire82/org.openhab.binding.heos/tree/HandlerFactoryTest